src/zkbob/ZkBobPool.sol

```diff
@@ -61,18 +61,46 @@ abstract contract ZkBobPool is IZkBobPool, EIP1967Admin, Ownable, Parameters, Ex
 
     mapping(address => uint256) public accumulatedFee;
 
+    // TODO:
+    // Is it safe to just add deprecated gap here and
+    // call setTokenSeller() again after upgrade to fix storage layout?
+    // Additionally, it makes sense to add __gap as well.
+    // address private __deprecatedGap2;
```

Comment:
Yes, this approach should work. Current token seller value will go to this slot and will stay unused, while we will re-initialize a new slot during the upgrade.

Reply:
Good, thank you! I've added gaps in [3d39ff4](https://github.com/zkBob/zkbob-contracts/pull/97/commits/3d39ff4c1844f8d064ac073b1ee3801b5749bbb4).